scripts/test_maps/test_maps.R

```diff
@@ -0,0 +1,68 @@
+# stand alone static map example
+# remotes::install_github("vlizbe/imis")
+# remotes::install_github("lifewatch/eurobis")
+
+library(eurobis)
+library(ggplot2)
+library('rnaturalearth')
+library(magick)
+
+# This test relies on produce_maps.R
+# Run the code inside the loop on L104 setting first ss = 1. DO NOT RUN THE LOOP ITSELF
+# Run from line 105 to 133
+# Here a saved version of r1 will be loaded
+r1 <- readRDS(file.path(".", "scripts", "test_maps", "r1.rds"))
+
+# Show countries
+world <- ne_countries(scale = "medium", returnclass = "sf")
+
+# EMODnet colors
+emodnetColor <- list(
+  # First palette
+  blue = "#0A71B4",
+  yellow = "#F8B334",
+  darkgrey = "#333333",
+  # Secondary palette,
+  darkblue = "#012E58",
+  lightblue = "#61AADF",
+  white = "#FFFFFF",
+  lightgrey = "#F9F9F9"
+)
+
+# EMODnet logo
+logo_raw <- image_read("https://www.emodnet-biology.eu/sites/emodnet-biology.eu/files/public/logos/logo-footer.png") 
+
+# Transform raster to vector
+grid <- sf::st_as_sf(raster::rasterToPolygons(r1))
+
+# Plot the grid
+plot_grid <- ggplot() +
+  geom_sf(data = world, 
+          fill = emodnetColor$darkgrey, 
+          color = emodnetColor$lightgrey, 
+          size = 0.1) +
+  geom_sf(data = grid, aes(fill = layer), size = 0.05) +
+  coord_sf(crs = 3035, xlim = c(2426378.0132, 7093974.6215), ylim = c(1308101.2618, 5446513.5222)) +
```

Comment:
@salvafern could you 'zoom' the map to the extent of the greater North Sea? I think it's not needed to show whole of Europe. You do that here

Reply:
Done. See commit [0ec37578a965a0f7b74b84ffe9743666febc9fda](https://github.com/salvafern/Benthos_greater_North_Sea/commit/0ec37578a965a0f7b74b84ffe9743666febc9fda)